Source/IGListAdapterUpdater.m

```diff
@@ -380,16 +380,22 @@ - (void)queueUpdateWithCollectionView:(UICollectionView *)collectionView {
     }
 
     __weak __typeof__(self) weakSelf = self;
+    __weak __typeof__(collectionView) weakCollectionView = collectionView;
+
+    // dispatch_async to give the main queue time to collect more batch updates so that a minimum amount of work
+    // (diffing, etc) is done on main. dispatch_async does not garauntee a full runloop turn will pass though.
+    // see -performUpdateWithCollectionView:fromObjects:toObjects:animated:]objectTransitionBlock:completion: for more
+    // details on how coalescence is done.
```

Comment:
Added this b/c I keep getting questions about this

Reply:
guarantee?

Reply:
I will never, ever be able to spell this word right on the first try.